### DISCUSSION TITLE
Error loading robot system

### DISCUSSION THREAD

**Discussion by baharmon** [visose/Robots#90] [Q&A] [answered]
Hi, I'm getting this error when trying to load robot system:

> 1. Solution exception:The 'Base' start tag on line 17 position 9 does not match the end tag of 'Positioner'. Line 24, position 9.

I believe I followed the installation instructions correctly. For installing the robot libraries, I tried downloading the repo and copying them. After reading issues 4 and 15, I also tried downloading the xml from raw view. Any ideas? 

Thanks, Brendan

**Comment by visose** [comment]
Hi Brendan, my guess is that you have a malformed XML.
Check that the "base" tag on line 17 is self closing, with a forward slash at the end, like this:
`<Base x="0.000" y="0.000" z="0.000" q1="1.000" q2="0.000" q3="0.000" q4="0.000"/>`
Could you attach it here?

**Comment by baharmon** [comment]
I've tried a new document and your simple example.gh using the Bartlett library. The error message is the same before I set a robot name, when I set robot name, or when I open the example. The line numbers in the error don't match the code in bartlett.xml.

[bartlett.zip](https://github.com/visose/Robots/files/4134166/bartlett.zip)

<img width="1403" alt="load-robot-system" src="https://user-images.githubusercontent.com/10392927/73454345-801d0500-4333-11ea-85d0-e836eb945e7b.png">
<img width="1403" alt="simple-example" src="https://user-images.githubusercontent.com/10392927/73454349-8317f580-4333-11ea-88cc-eb4b80a28328.png">

**Comment by baharmon** [comment, marked as answer]
Ah, I forgot to unblock the rhino model bartlett.3dm. Now it works. Thank you for the help.

**Comment by visose** [comment]
Great 👍

**Comment by cfboyd** [comment]
Hi I'm having the same error but I did actually unblock bartlett.3dm. I've also downloaded grymsdyke to see if the bartlett file was the issue, and I can see the grymsdyke robots in the dropdown menu but i still get that error with the bartlett ones, the grymsdyke ones, and the default robots.

**Comment by cfboyd** [comment]
Ignore this. The issue ended up being that grasshopper was looking in my user/onedrive/documents rather than user/documents for the Robots folder
